### ISSUE TITLE
Movies from Radarr v3 don't show up in the calendar on the homepage

### ISSUE THREAD

**Issue by daften** [causefx/Organizr#1480]
###### Operating System: Docker image (official)
<hr>

##### Problem Description:

I've added the radarr integration for the homepage, and enabled it. When testing the connection, it works. But when checking the calendar on the homepage, movies from the radarr calendar aren't put in the calendar on the homescreen.

When searching media, the movies do show up.

<hr>

##### Reproduction Steps:

Very specific:
* Add radarr to a docker-compose file, like in htpcBeginner/traefik
* Add organizr
* Point organizr to the radarr docker image with internal hostname, and enable the connection
* Check the calendar on the homepage

<hr>

#### Errors on screen?  If so paste here:

```

```

**Comment by juliemars** [comment]
same issue for me just upgraded to Radarr V3 running on Win10 and the movies have disappeared from the calendar. 

Organizr is running in Docker under 
organizr/organizr:latest branch=dev

connection tests fine

I don't see anything in the logs to show an issue

**Comment by causefx** [comment]
is radarr the only item enabled atm or is sonarr etc?

**Comment by daften** [comment]
Sonarr is enabled too, but disabling that didn't seem to help for me

**Comment by juliemars** [comment]
same disabled Sonarr and Lidarr no difference.
disabled sonarr, Lidarr and Radarr nothing showing confirmed nothing shows.
Added as an ical under calendar but nothing shows for radarr, sonarr works fine.

**Comment by juliemars** [comment]
OK so I have done some more checks and I don't think this is an Organizr issue I added the calendar for V3 to Outlook and nothing shows. It must be a Radarr issue @daften can you check if this is the same for you? if it is we will need to post on Radarr issues.

**Comment by HalianElf** [comment]
Make sure you're looking at physical releases. It doesn't pull cinema releases.

**Comment by juliemars** [comment]
@HalianElf ah that is starting to make sense, they are all tagged as Digital bar which I haven't seen before. 

I wonder if this is a new tag and the Calendar needs tweaked to include it in the link

**Comment by HalianElf** [comment]
Can you give an example of one of the ones you have on your radarr calendar so I can look at the flags on it?

**Comment by juliemars** [comment]
no problem, they are all hallmark Christmas movies unfortunately 
![image](https://user-images.githubusercontent.com/17718528/97773630-5fdfd800-1b49-11eb-9c84-cfc2e45ace48.png)
![image](https://user-images.githubusercontent.com/17718528/97773635-6cfcc700-1b49-11eb-87be-e746a16eb667.png)
![image](https://user-images.githubusercontent.com/17718528/97773646-8271f100-1b49-11eb-9466-61bc272330ad.png)

**Comment by HalianElf** [comment]
Thanks, will check on these later against my PR later.

**Comment by HalianElf** [comment]
@juliemars if you're on the v2-develop branch, try out the new toggles and see if it fixes it

**Comment by juliemars** [comment]
@HalianElf spotted it was in the latest update and was trying it, it did take a few refreshes and goes before I got it working. I first ticked all 3 toggles that were not selected and hit save. 

On going back to the homepage and hitting refresh I ended up with no calendar at all. Multiple refreshes and nothing loaded. 
I then disabled Radarr fully and the calendar returned.

I then enabled the options 1 at a time and same thing happened, but once all were enabled and I disabled and reenabled Radarr it finally showed up - not sure if it's an issue with my setup or a code issue but I can see them now:

![image](https://user-images.githubusercontent.com/17718528/98037477-8b95e300-1e13-11eb-9086-e81589f9e378.png)
![image](https://user-images.githubusercontent.com/17718528/98037488-905a9700-1e13-11eb-8d29-eb29f25a903e.png)

Thanks for looking into this and fixing it.
